### PR TITLE
[ZT] Update Do Not Inspect guidance

### PR DESCRIPTION
**Pull request by maxvp**

### CODE REVIEW

**Review by kodster28 — APPROVED** [review]

File: content/cloudflare-one/tutorials/do-not-decrypt.md [hunk 1]
```diff
@@ -10,37 +10,35 @@ title: Skip inspection for groups of applications
...
 5 minutes
 
-## Before you start
-
-[Enable HTTP filtering](/cloudflare-one/policies/filtering/initial-setup/http/).
-
 ## Build the policy
 
-1.  Navigate to the **HTTP** tab of the `Policies` page in Cloudflare Gateway. Click **Add a rule**.
+{{<Aside>}}
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/do-not-decrypt.md]: We've been using the Prerequisites header in ZT docs. Example: https://developers.cloudflare.com/cloudflare-one/tutorials/azuread-risky-users/

File: content/cloudflare-one/tutorials/do-not-decrypt.md [hunk 2]
```diff
@@ -10,37 +10,35 @@ title: Skip inspection for groups of applications
...
 
-1.  Under **Selector** choose **Application**. Select **in** in the **Operator field**. In the **Value** field, select the applications you wish to include.
+3. Under **Selector**, choose **Application**. In the **Operator** field, select **in**. In the **Value** field, select the applications you wish to include.
 
-1.  Scroll to the bottom of the page and select **Do Not Inspect** in the `Select an action` section, then click **Create rule**.
+4.  In **Select an action**, select **Do Not Inspect**. Select **Create policy**.
 
 ## Change rule precedence
 
 New rules are saved at the bottom of the rule list in Gateway. Gateway evaluates rules from top-to-bottom, except for do-not-inspect rules. Those are always evaluated first.
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/do-not-decrypt.md]: can link to https://developers.cloudflare.com/cloudflare-one/policies/filtering/order-of-enforcement/

File: content/cloudflare-one/tutorials/do-not-decrypt.md [hunk 3]
```diff
@@ -10,37 +10,35 @@ title: Skip inspection for groups of applications
 
 You can configure Cloudflare Zero Trust to skip inspection for certain groups of applications.
 
-By default, Cloudflare Gateway creates a rule that includes the hostnames used by certain client applications, like Zoom or Apple's services, that rely on certificate pinning. The TLS inspection performed by a service like Cloudflare Gateway will cause errors when users visit those applications.
-
-This tutorial skips inspection for additional applications beyond those in the list curated by Cloudflare.
+Certain client applications, such as Zoom or Apple services, rely on certificate pinning. The TLS inspection performed by Cloudflare Gateway will cause errors when users visit those applications. To avoid this behavior, you must add a Do Not Inspect HTTP policy.
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/do-not-decrypt.md]: can link to https://developers.cloudflare.com/cloudflare-one/policies/filtering/http-policies/tls-decryption/

File: content/cloudflare-one/tutorials/do-not-decrypt.md [hunk 4]
```diff
@@ -10,37 +10,35 @@ title: Skip inspection for groups of applications
...
+Before you start, you must [enable HTTP filtering](/cloudflare-one/policies/filtering/initial-setup/http/).
+{{</Aside>}}
+
+1. In the [Zero Trust dashboard](https://one.dash.cloudflare.com), go to **Gateway** > **Firewall Policies** > **HTTP**. Select **Create a policy**.
 
-1.  Name the policy and, optionally, provide a description.
+2.  Name the policy and, optionally, provide a description.
 
-1.  Under **Selector** choose **Application**. Select **in** in the **Operator field**. In the **Value** field, select the applications you wish to include.
+3. Under **Selector**, choose **Application**. In the **Operator** field, select **in**. In the **Value** field, select the applications you wish to include.
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/do-not-decrypt.md]:
```suggestion
3. Under **Selector**, choose **Application**. In the **Operator** field, select **in**. In the **Value** field, search for `Do Not Inspect` and select the applications you wish to include.
```

**Review by ranbel — APPROVED** [review]

**Pull request merged**